sdk/resourcemanager/docs/SINGLE_SERVICE_PACKAGES.md

```diff
@@ -751,6 +752,7 @@ service
 * [package-preview-2022-12](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/elasticsan/resource-manager#tag-package-preview-2022-12)
     * [1.0.0-beta.2](https://repo1.maven.org/maven2/com/azure/resourcemanager/azure-resourcemanager-elasticsan/1.0.0-beta.2)
 * [package-2023-01](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/elasticsan/resource-manager#tag-package-2023-01)
+    * [1.0.0](https://repo1.maven.org/maven2/com/azure/resourcemanager/azure-resourcemanager-elasticsan/1.0.0)
```

Comment:
Seems like the sort should ignore tags and only sort the version. Will do it in following PR.

Reply:
PR: https://github.com/Azure/azure-sdk-for-java/pull/38512